dbt_transformation_create_test.go

```diff
@@ -0,0 +1,48 @@
+package fivetran_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/fivetran/go-fivetran"
+)
+
+func TestNewDbtTransformationCreateE2E(t *testing.T) {
+	t.Skip("Skipping test until we get more info on dbt transformations data which can be used for testing")
```

Comment:
We could introduce mocked tests now, this test doesn't name any sense, it's just a code that won't be executed.

Reply:
Done :)